packages/ui/postcss.config.js

```diff
@@ -25,7 +25,7 @@ Infinite Reality Engine. All Rights Reserved.
 */
 
 
-module.exports = {
+export default {
```

Comment:
need to revert this as we reverted using ESM

Reply:
reverted